webflow-designer-extension-cli/public/styles.css

```
@@ -343,89 +441,222 @@ select {
+
+.icon--arrow--left {
+  transform: rotate(180deg);
+}
+
+.icon--arrow--up {
+  transform: rotate(270deg);
+}
+
+.corners--all {
+  border-radius: var(--radius--notch);
+  corner-shape: notch;
```

Biome (2.4.4)

[error] Unknown property is not allowed.

(lint/correctness/noUnknownProperty)